libs/api/domains/air-discount-scheme/src/lib/api-domains-air-discount-scheme.resolver.ts

```diff
@@ -0,0 +1,62 @@
+import { UseGuards } from '@nestjs/common'
+import { Query } from '@nestjs/graphql'
+import type { User } from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+import { AirDiscountSchemeService } from './api-domains-air-discount-scheme.service'
+
+import {
+  Discount as TDiscount,
+  User as TUser,
+} from '@island.is/air-discount-scheme/types'
+import { Discount } from '../models/discount.model'
+
+type DiscountWithTUser = Discount & { user: TUser }
+
+const TWO_HOURS = 7200
+
+@UseGuards(IdsAuthGuard, IdsUserGuard, ScopesGuard)
```

Comment:
You don't need both `IdsAuthGuard` and `IdsUserGuard`. According to your resolver you should use the `IdsUserGuard` and remove the `IdsAuthGuard`.

Reply:
Good catch, thank you